fendermint/eth/api/examples/ethers.rs

```diff
@@ -405,7 +405,8 @@ where
     // However, ethers.js actually asserts this and we cannot disable it, rendering that, or any similar tool, unusable if we rely on
     // the default Tendermint transaction hash, which is a Sha256 hash of the entire payload (which includes the signature),
     // not a Keccak256 of the unsigned RLP.
-    assert_eq!(tx_hash, transfer.sighash(), "Ethereum hash should match");
+    // let expected_hash = todo!("we need to reproduce the signature to get the full RLP");
```

Comment:
Is this on purpose?

Reply:
Yeah, I didn't want to go into the trouble of using the signer middleware to produce a signature which I can recombine into a signed RLP, but that was out of laziness, I will check now if it would be simple to do.

Reply:
Done, thanks for keeping me honest here :pray: